website/docs/r/network_connection_monitor.html.markdown

```diff
@@ -74,41 +157,131 @@ The following arguments are supported:
 
 * `name` - (Required) The name which should be used for this Network Connection Monitor. Changing this forces a new Network Connection Monitor to be created.
 
-* `resource_group_name` - (Required) The name of the Resource Group where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
-
 * `location` - (Required) The Azure Region where the Network Connection Monitor should exist. Changing this forces a new Network Connection Monitor to be created.
 
-* `destination` - (Required) A `destination` block as defined below.
+* `network_watcher_id` - (Required) The ID of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
 
-* `network_watcher_name` - (Required) The name of the Network Watcher. Changing this forces a new Network Connection Monitor to be created.
+* `endpoint` - (Required) A `endpoint` block as defined below.
 
-* `source` - (Required) A `source` block as defined below.
+* `test_configuration` - (Required) A `test_configuration` block as defined below.
+
+* `test_group` - (Required) A `test_group` block as defined below.
 
 ---
 
-* `auto_start` - (Optional) Will the connection monitor start automatically once created? Changing this forces a new Network Connection Monitor to be created.
+* `notes` - (Optional) Any notes about the Network Connection Monitor.
```

Comment:
Can we modify this statement a bit?

Reply:
This sentence is copied from existing resource.

Reply:
Then could we fix this statement and apply the updated improved versinoit to where it was copied from?

Reply:
Sure. Updated the description for this property. For the others, I will update them in another separate PR.